CHANGES/4970.misc

```diff
@@ -0,0 +1 @@
+Adding mazer_url to AnsibleDistribution
```

Comment:
I usually leave newlines at the end of files in case they are ever concatenated together.